css-scroll-snap-2/Overview.bs

```diff
@@ -279,13 +279,35 @@ Snap Events {#snap-events}
 				<th>Event handler event type
 		<tbody>
 			<tr>
-				<th><dfn event>snapChanged</dfn>
+				<th><dfn event>snapchanged</dfn>
 				<td>{{scroll!!event}}
 			<tr>
-				<th><dfn event>snapChanging</dfn>
+				<th><dfn event>snapchanging</dfn>
 				<td>{{scroll!!event}}
 	</table>
 
+	{{ snapchanged }} will indicate that the set of snap areas to which a snap container is snapped has changed.
+	{{ snapchanging }} will hint at the set of snap areas to which a snap container would snap during a scroll.
+	{{ snapchanged }} and {{ snapchanging }} events will implement {{ SnapEvent}}.
+
+
+SnapEvent interface
+-------------------
+
+<pre class="idl">
+			[Exposed=Window]
+			interface SnapEvent : Event {
+				readonly attribute NodeList snapTargets;
+			};
+</pre>
+
+<dl>
+		<dt><code>SnapEvent . snapTargets</code></dt>
+		<dd>
+			A list of elements that a scroller is considered snapped to on either axis.
```

Comment:
This could use more details. In particular,

What is the order of the elements?

If you look at [getAnimations](https://drafts.csswg.org/web-animations-1/#dom-animatable-getanimations) it says:
> The returned list is sorted using the composite order described for the associated [animations](https://drafts.csswg.org/web-animations-1/#concept-animation) of effects in [§ 5.4.2 The effect stack](https://drafts.csswg.org/web-animations-1/#the-effect-stack).

Where the composite order is defined in:
https://drafts.csswg.org/web-animations-1/#animation-composite-order

I suggest defining an order similar to the [css animation composite order](https://drafts.csswg.org/css-animations-2/#animation-composite-order) which is basically [tree order](https://dom.spec.whatwg.org/#concept-tree-order) with a defined ordering for pseudo-elements.

What does it mean that a scroller is considered snapped to a particular element?

For this, you probably want to refer to being in a valid snap position for the particular element where this is defined in css-scroll-snap-1. You should also mention how for snapchanging it is referring to the eventual scroll position, not the position that is currently scrolled to.

Reply:
Thanks for the links. I think it suffices to just specify that the list is sorted in tree order so I've included that comment. One point of consideration was pseudo-elements: tree order does not seem to cover pseudo-elements, but since there are no elements for pseudo-elements, pseudo elements would not be included in snapTargets which is a list of elements.

I have also added a reference to the css-scroll-snap-1 spec for clarity on the criteria for populating snapTargets.